src/langchain_google_alloydb_pg/async_vectorstore.py

```diff
@@ -534,13 +534,19 @@ async def __query_collection(
         embedding: list[float],
         k: Optional[int] = None,
         filter: Optional[str] = None,
+        columns: Optional[list[str]] = None,
```

Comment:
Instead of changing this method signature and duplicating the code to create the column list, just do that from this method